src/uct/tcp/tcp_iface.c

```diff
@@ -213,15 +213,27 @@ static int uct_tcp_iface_is_reachable(const uct_iface_h tl_iface,
     return 1;
 }
 
-static const char *
-uct_tcp_iface_get_sysfs_path(const char *dev_name, char *path_buffer)
+static const char *uct_tcp_iface_get_sysfs_path(const char *dev_name,
+                                                char *path_buffer,
+                                                size_t path_buffer_size)
 {
-    char dev_path[PATH_MAX];
+    char dev_path[path_buffer_size];
```

Comment:
ok, seems no point to pass path_buffer_size. because uct_iface_get_sysfs_path uses it for realpath() that already assumes its size is PATH_MAX. so maybe just need to document the size of path_buffer is PATH_MAX.